npm run build triggers error

### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/boyney123/eventcatalog/blob/master/CONTRIBUTING.md#reporting-new-issues).

### Description

When running `npm run build` in the root of the project, this error occurs.

```bash
$ tsc
error TS5055: Cannot write file '/Users/Sites/eventcatalog/packages/eventcatalog-plugin-generator-amazon-eventbridge/lib/index.d.ts' because it would overwrite input file.
error TS5055: Cannot write file '/Users/Sites/eventcatalog/packages/eventcatalog-plugin-generator-amazon-eventbridge/lib/lib/aws.d.ts' because it would overwrite input file.
error TS5055: Cannot write file '/Users//eventcatalog/packages/eventcatalog-plugin-generator-amazon-eventbridge/lib/markdown.d.ts' because it would overwrite input file.
error TS5055: Cannot write file '/Users/tes/eventcatalog/packages/eventcatalog-plugin-generator-amazon-eventbridge/lib/types.d.ts' because it would overwrite input file.
```


### Steps to reproduce

1. goto the root folder of the project
2. yarn install
3. npm run build

### Expected behavior

build runs as expected, so that we can compile the asyncapi-plugin.

### Actual behavior

When running `npm run build` in the root of the project, this error occurs.

```bash
$ tsc
error TS5055: Cannot write file '/Users/Sites/eventcatalog/packages/eventcatalog-plugin-generator-amazon-eventbridge/lib/index.d.ts' because it would overwrite input file.
error TS5055: Cannot write file '/Users/Sites/eventcatalog/packages/eventcatalog-plugin-generator-amazon-eventbridge/lib/lib/aws.d.ts' because it would overwrite input file.
error TS5055: Cannot write file '/Users//eventcatalog/packages/eventcatalog-plugin-generator-amazon-eventbridge/lib/markdown.d.ts' because it would overwrite input file.
error TS5055: Cannot write file '/Users/tes/eventcatalog/packages/eventcatalog-plugin-generator-amazon-eventbridge/lib/types.d.ts' because it would overwrite input file.
```


### Your environment

- EventCatalog version used:  0.0.1
- Environment name and version (e.g. Chrome 89, Node.js 16.4): NodeJS 17.0
- Operating system and version (e.g. Ubuntu 20.04.2 LTS): Mac Big Sur

--- a/packages/eventcatalog-plugin-generator-amazon-eventbridge/tsconfig.json
+++ b/packages/eventcatalog-plugin-generator-amazon-eventbridge/tsconfig.json
@@ -13,5 +13,5 @@
     "skipLibCheck": false,
     "incremental": true
   },
-  "exclude": ["./scripts/**/*", "*/__tests__/**/*"]
+  "exclude": ["./scripts/**/*", "*/__tests__/**/*", "./lib/**"]
 }

--- a/packages/eventcatalog-plugin-generator-amazon-eventbridge/tsconfig.json
+++ b/packages/eventcatalog-plugin-generator-amazon-eventbridge/tsconfig.json
@@ -13,5 +13,5 @@
     "skipLibCheck": false,
     "incremental": true
   },
-  "exclude": ["./scripts/**/*", "*/__tests__/**/*", "./lib/**"]
+  "exclude": ["./scripts/**/*", "*/__tests__/**/*", "./lib/**/*"]
 }

--- a/packages/eventcatalog-plugin-generator-asyncapi/tsconfig.json
+++ b/packages/eventcatalog-plugin-generator-asyncapi/tsconfig.json
@@ -12,5 +12,5 @@
     "skipLibCheck": false,
     "incremental": true
   },
-  "exclude": ["./scripts/**/*", "*/__tests__/**/*"]
+  "exclude": ["./scripts/**/*", "*/__tests__/**/*", "./lib/**"]
 }

--- a/packages/eventcatalog-plugin-generator-asyncapi/tsconfig.json
+++ b/packages/eventcatalog-plugin-generator-asyncapi/tsconfig.json
@@ -12,5 +12,5 @@
     "skipLibCheck": false,
     "incremental": true
   },
-  "exclude": ["./scripts/**/*", "*/__tests__/**/*", "./lib/**"]
+  "exclude": ["./scripts/**/*", "*/__tests__/**/*", "./lib/**/*"]
 }

